registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MessageBusAddress.java

```diff
@@ -1,215 +1,226 @@
+	/** The Constant REGISTRATION_CONNECTOR_BUS_OUT. */
+	public static final MessageBusAddress REGISTRATION_CONNECTOR_BUS_OUT = new MessageBusAddress(
+			"registration-connector-bus-out");
+
+	/** The Constant PRINTING_BUS_IN. */
+	public static final MessageBusAddress PRINTING_BUS = new MessageBusAddress("printing-bus-in");
+
+	/** The Constant PRINTING_BUS_RESEND. */
+	public static final MessageBusAddress PRINTING_BUS_RESEND = new MessageBusAddress("printing-bus-resend");
+
+	/** The Constant EXTERNAL_STAGE_BUS_IN. */
+	public static final MessageBusAddress EXTERNAL_STAGE_BUS_IN = new MessageBusAddress("external-bus-in");
+
+	/** The Constant EXTERNAL_STAGE_BUS_OUT. */
+	public static final MessageBusAddress EXTERNAL_STAGE_BUS_OUT = new MessageBusAddress("external-bus-out");
+
+	/** The Constant ABIS_MIDDLEWARE_BUS_IN. */
+	public static final MessageBusAddress ABIS_MIDDLEWARE_BUS_IN = new MessageBusAddress("abis-middle-ware-bus-in");
+
+	/** The Constant ABIS_MIDDLEWARE_BUS_OUT. */
+	public static final MessageBusAddress ABIS_MIDDLEWARE_BUS_OUT = new MessageBusAddress("abis-middle-ware-bus-out");
+
+	/** The Constant ABIS_HANDLER_BUS_IN. */
+	public static final MessageBusAddress ABIS_HANDLER_BUS_IN = new MessageBusAddress("abis-handler-bus-in");
+
+	/** The Constant ABIS_HANDLER_BUS_OUT. */
+	public static final MessageBusAddress ABIS_HANDLER_BUS_OUT = new MessageBusAddress("abis-handler-bus-out");
+
+	/** The Constant BIOMETRIC_AUTHENTICATION_BUS_IN. */
+	public static final MessageBusAddress BIOMETRIC_AUTHENTICATION_BUS_IN = new MessageBusAddress("biometric-authentication-bus-in");
+
+	/** The Constant BIOMETRIC_AUTHENTICATION_BUS_OUT. */
+	public static final MessageBusAddress BIOMETRIC_AUTHENTICATION_BUS_OUT = new MessageBusAddress("biometric-authentication-bus-out");
+
+	/** The Constant QUALITY_CHECKER_BUS_IN. */
+	public static final MessageBusAddress QUALITY_CHECKER_BUS_IN = new MessageBusAddress("quality-checker-bus-in");
+
+	/** The Constant QUALITY_CHECKER_BUS_OUT. */
+	public static final MessageBusAddress QUALITY_CHECKER_BUS_OUT = new MessageBusAddress("quality-checker-bus-out");
+
+	/** The Constant SECUREZONE_NOTIFICATION_IN. */
+	public static final MessageBusAddress SECUREZONE_NOTIFICATION_IN = new MessageBusAddress("securezone-notification-bus-in");
+
+	/** The Constant SECUREZONE_NOTIFICATION_OUT. */
+	public static final MessageBusAddress SECUREZONE_NOTIFICATION_OUT = new MessageBusAddress("securezone-notification-bus-out");
+
+	/** The Constant STRUCTURE_BUS_IN. */
+	public static final MessageBusAddress PACKET_CLASSIFIER_BUS_IN = new MessageBusAddress("packet-classifier-bus-in");
+
+	/** The Constant STRUCTURE_BUS_OUT. */
+	public static final MessageBusAddress PACKET_CLASSIFIER_BUS_OUT = new MessageBusAddress("packet-classifier-bus-out");
+	
+	/** The Constant WORKFLOW_EVENTUPDATE_BUS_IN. */
+	public static final MessageBusAddress WORKFLOW_EVENTUPDATE_BUS_IN = new MessageBusAddress(
+			"workflow-eventupdate-bus-in");

```

Comment:
eventupdate is two different words, any paticular reason it is not seperated with hyphen ?

Reply:
Also lets remove bus-in, since workflow event update is not a stage and it will never have a bus out

Reply:
I will fix it